src/utils/debugdraw_gl.cc

```diff
@@ -10,54 +10,13 @@
 #include <epoxy/gl.h>
 #include <glm/glm.hpp>
 #include <glm/ext.hpp>
+#include "../shader.h"
 
+#define DEBUG_DRAW_IMPLEMENTATION
+#define DEBUG_DRAW_CXX11_SUPPORTED 1
+#define DEBUG_DRAW_USE_STD_MATH 1
```

Comment:
I think STD_MATH can go away now? You removed check around the bits it was guarding.